### PR TITLE
Clarify task descriptions in README

### PR DESCRIPTION
**Pull request by brunobowden**

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @confile - PTAL

File: README.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ These are the main tasks for the plugin:
 
     j2objcCycleFinder       - Find cycles that can cause memory leaks, see notes below
     j2objcTranslate         - Translates to Objective-C
-    j2objcAssemble          - Builds debug/release libraries, packs targets in to fat libraries
-    j2objcTest              - Runs all JUnit tests, as translated into Objective-C
-    j2objcBuild             - Builds and tests all j2objc outputs.
-    j2objcXcode             - Configure Xcode to link static library & header files, uses CocoaPods
+    j2objcAssemble          - Outputs packed libraries, source and resources to j2objcOutputs
```
Comment (advayDev1) [inline README.md]: ... build/j2objcOutputs ///

File: README.md [hunk 2]
```diff
@@ -93,10 +93,10 @@ These are the main tasks for the plugin:
...
+    j2objcTest              - Runs all JUnit tests on translated code
```
Comment (advayDev1) [inline README.md]: it is running all the tests in translated code, not on.  in the sense that the tests too are translated

**Comment by advayDev1** [comment]
LGTM with comments

File: README.md (hunk 2 above)
Comment (brunobowden) [inline README.md]: Updated to:

    j2objcTest              - Runs all JUnit tests in the translated code

File: README.md (hunk 1 above)
Comment (brunobowden) [inline README.md]: Updated to:

    j2objcAssemble          - Outputs packed libraries, source & resources to build/j2objcOutputs

**Pull request merged**